src/lib/routes/admin-api/context.ts

```diff
@@ -65,6 +65,9 @@ export class ContextController extends Controller {
             middleware: [
                 openApiService.validPath({
                     tags: ['Context'],
+                    summary: 'Gets configured context fields',
+                    description:
+                        'Returns all configured [Context fields](https://docs.getunleash.io/how-to/how-to-define-custom-context-fields) that have been created.',
```

Comment:
This is a little unclear: Can a context field be configured and not created? What about the other way: can it be created and not configured?